packages/electron-updater/src/DebUpdater.ts

```diff
@@ -29,10 +29,8 @@ export class DebUpdater extends BaseUpdater {
 
   protected doInstall(options: InstallOptions): boolean {
     const sudo = this.wrapSudo()
-    // pkexec doesn't want the command to be wrapped in " quotes
-    const wrapper = /pkexec/i.test(sudo) ? "" : `"`
     const cmd = ["dpkg", "-i", options.installerPath, "||", "apt-get", "install", "-f", "-y"]
-    this.spawnSyncLog(sudo, [`${wrapper}/bin/bash`, "-c", `'${cmd.join(" ")}'${wrapper}`])
+    this.spawnSyncLog(sudo, cmd)
```

Comment:
Imo, this way of installing local `.deb` in 2 steps with `dpkg` and `apt-get install -f` can be updated. Apt is now able to install local `.deb` and depends in a single command if the package argument is an absolute path to a `.deb`.

You may now do `this.spawnSyncLog("sudo", ["apt-get", "install",  "-y", options.installerPath])`. This will avoid you to pipe commands, or to split it into 2 commands and checking return code.